workers/ohsome_quality_analyst/base/indicator.py

```diff
@@ -87,6 +85,39 @@ def __init__(
 
         self._validate_indicator_layer(self.__class__.__name__, layer_name)
 
+    def as_feature(self) -> Feature:
+        """Returns a GeoJSON Feature object.
+
+        The properties of the Feature contains the attributes of the indicator.
+        The geometry (and properties) of the input GeoJSON object is preserved.
+        """
+        result = vars(self.result).copy()
+        result.pop("svg")
+        # Prefix all keys of the dictionary
+        result = {"result." + str(key): val for key, val in result.items()}
+        data = {"data." + str(key): val for key, val in self.data.items()}
+        geometry = self.feature.geometry
+        properties = {
+            "metadata.name": self.metadata.name,
+            "metadata.description": self.metadata.description,
+            "layer.name": self.layer.name,
+            "layer.description": self.layer.description,
+            **result,
+            **data,
+            **self.feature.properties,
+        }
```

Comment:
Personally, I would integrate the dictionaries to unpack directly in the properties environment:
```suggestion
        geometry = self.feature.geometry
        properties = {
            "metadata.name": self.metadata.name,
            "metadata.description": self.metadata.description,
            "layer.name": self.layer.name,
            "layer.description": self.layer.description,
            # Prefix all keys of the dictionary
            **{"result." + str(key): val for key, val in result.items()},
            **{"data." + str(key): val for key, val in self.data.items()},
            **self.feature.properties,
        }
```
If you don't want to do that, I would at least not overwrite the result variable:
```suggestion
        # Prefix all keys of the dictionary
        result_properties = {"result." + str(key): val for key, val in result.items()}
        data_properties = {"data." + str(key): val for key, val in self.data.items()}
        geometry = self.feature.geometry
        properties = {
            "metadata.name": self.metadata.name,
            "metadata.description": self.metadata.description,
            "layer.name": self.layer.name,
            "layer.description": self.layer.description,
            **result_properties,
            **data_properties,
            **self.feature.properties,
        }
```

Reply:
Implemented your first suggestion.